tf-modules/ingest/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@cumulus/ingest-module",
+  "version": "3.0.0",
+  "description": "Terraform module for data ingest related functionality",
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "scripts": {
+    "prepare": "node scripts/write-cumulus-version.js"
```

Comment:
Double checking - this is solely for this lifecycle script?

Reply:
I'm not sure what you mean. This is just here so that we have a hook to ensure that the cumulus_version.json file gets generated before deployment, since message_template.tf refers to that file

Reply:
Sorry, the package.json is solely for the prepare hook.   Sounds like yes, so 👍